libs/content/workspace/src/tab/markdown_editor/galleys.rs

```diff
@@ -88,15 +82,7 @@ pub fn calc(
             let maybe_link_range = link_idx.map(|link_idx| bounds.links[link_idx]);
             let in_selection = selection_idx.is_some() && !buffer.cursor.selection.is_empty();
 
-            let captured = bounds::captured(
-                buffer.cursor,
-                &bounds.paragraphs,
-                ast,
-                text_range,
-                hover_syntax_reveal_debounce_state,
-                appearance,
-                cursor_paragraphs,
-            );
+            let captured = text_idx.is_none();
```

Comment:
Here, we leverage the capture calculation done as part of figuring out what text is rendered instead of redoing the work. This simplifies the code by keeping character capture logic in one place.